test/benchmark.hs

```diff
@@ -1,7 +1,15 @@
 {-
-  Install microbench to build this program:
+  This program should generally be run using `cabal bench` or
+  `stack bench`. To use `stack bench`, edit stack.yaml to include
+
+  extra-deps:
+  - microbench-0.1
+
```

Comment:
I don't think we ship a `stack.yaml`, and I don't know enough about `stack` to feel comfortable doing so. But developers hacking on `fgl` can very easily write their own, and I don't think it can hurt to give a bit of guidance to help them.